include/node.hpp

```diff
@@ -253,12 +253,13 @@ class node : protected std::tuple<Arguments...> {
     using derived_t                = Derived;
     using node_template_parameters = meta::typelist<Arguments...>;
     using Description              = typename node_template_parameters::template find_or_default<is_doc, EmptyDoc>;
-    constexpr static tag_propagation_policy_t                                                                      tag_policy  = tag_propagation_policy_t::TPP_ALL_TO_ALL;
-    A<uint64_t, "numerator", Doc<"decimation/interpolation settings">>                                             numerator   = 1_UZ;
-    A<uint64_t, "denominator", Doc<"decimation/interpolation settings">>                                           denominator = 1_UZ;
-    A<uint64_t, "stride", Doc<"stride doc">>                                                                       stride      = 1_UZ;
-    const std::size_t                                                                                              unique_id   = _unique_id_counter++;
-    const std::string                                                                                              unique_name = fmt::format("{}#{}", fair::meta::type_name<Derived>(), unique_id);
+    constexpr static tag_propagation_policy_t                                                                      tag_policy     = tag_propagation_policy_t::TPP_ALL_TO_ALL;
+    A<std::size_t, "numerator", Doc<"decimation/interpolation settings">>                                          numerator      = 1_UZ;
```

Comment:
`uint64_t` or `uint32_t` would be better due to being more explicit w.r.t. portability than `std::size_t` (which is only vaguely defined).

Could you add the distilled doc description of the PR to the definitions here... This would help the users how to express/use these parameters and we wouldn't have to touch this again later. Thanks.

Reply:
I agree with your comment and I even implemented at first as `uint64_t`. But then I realized that all the rest parameters like port.min/max_samples, available_samples etc. are all `std::size_t` thus I decided to be consistent and also use `std::size_t`. What do you think?

Reply:
This is OK. Since these settings parameters may be set by other architectures/transmitted via the network we should opt for more portable types. This could be done when we tackle this: https://github.com/fair-acc/graph-prototype/issues/81